### PR TITLE
Adds "Edit this Doc" button for each document

### LINKED ISSUE #2578: Add Edit on GitHub link to documentation
**Is your feature request related to a problem or area of OpenRefine? Please describe.**

On pages like https://docs.openrefine.org/operations/key_value_columnize.html, it would be good to have an edit link, pointing to:
https://github.com/OpenRefine/OpenRefine/edit/master/docs/src/operations/key_value_columnize.md

**Describe the solution you'd like**
Integrate it nicely in the design (there might be native things in Docusaurus for that).

### PR DESCRIPTION
**Pull request by thadguidry**
closes #2578

### CODE REVIEW

File: docs/siteConfig.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ const siteConfig = {
 
   // URL for editing docs, usage example: editUrl + 'en/doc1.md'.
   // If this field is omitted, there will be no "Edit this Doc" button for each document.
-  editUrl: 'https://github.com/OpenRefine/OpenRefine/master/docs/',
+  editUrl: 'https://github.com/OpenRefine/OpenRefine/edit/master/docs/src/',
```
Comment (thadguidry) [inline docs/siteConfig.js:59]: I pasted the wrong one. thanks, just saw that also.

**Comment by thadguidry** [comment]
Yeah Netlify preview shows it's working!

**Comment by kushthedude** [comment]
@wetneb @thadguidry Shouldn't we redirect them just to the GitHub Document Page rather than redirecting to edit link?

BTW, Netlify is coming quite handy I presume 😄 .

**Comment by thadguidry** [comment]
@kushthedude yes, quite liking Netlify so far.  But not quite sure what you mean about the Page.  Which one are you speaking of?  What would you put as the `editURL` in Docusaurus `siteConfig.js`?

**Comment by kushthedude** [comment]
Like showing the button name as ` Edit on GitHub` and just redirect the
user to GitHub Source Link of the doc without /edit parameter in that.
As with the edit parameter GitHub will create a fork for the new users
which may sometimes not be the thing they wanted.
It is just a suggestion if seems appropriate else the current one is fine
too.

**Comment by thadguidry** [comment]
@kushthedude Sure, why not do a PR and we can see what it looks like and how it works and compare?

**Comment by wetneb** [comment]
> As with the edit parameter GitHub will create a fork for the new users
> which may sometimes not be the thing they wanted.

It only does if they actually save their changes, no? For instance go to this link:
https://github.com/apache/spark/edit/master/docs/cloud-integration.md
This will not create a fork of Spark on your side just by visiting this link - you need to save changes for that.

**Comment by lisa761** [comment]
@wetneb That link actually redirects us to this:

**Comment by wetneb** [comment]
Ah sorry I did not realize I already had a fork for that repository! Thanks!

**Pull request merged**